_vagrant/salt/pillar/platforms-saml-identity-provider.sls

```diff
@@ -0,0 +1,147 @@
+            ),
+            1 =>
+            array (
+              'index' => 1,
+              'Binding' => 'urn:oasis:names:tc:SAML:1.0:profiles:browser-post',
+              'Location' => 'http://192.168.120.60/module.php/saml/sp/saml1-acs.php/default-sp',
+            ),
+            2 =>
+            array (
+              'index' => 2,
+              'Binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Artifact',
+              'Location' => 'http://192.168.120.60/module.php/saml/sp/saml2-acs.php/default-sp',
+            ),
+            3 =>
+            array (
+              'index' => 3,
+              'Binding' => 'urn:oasis:names:tc:SAML:1.0:profiles:artifact-01',
+              'Location' => 'http://192.168.120.60/module.php/saml/sp/saml1-acs.php/default-sp/artifact',
+            ),
+          ),
+          'contacts' =>
+          array (
+            0 =>
+            array (
+              'emailAddress' => 'root@localhost',
+              'contactType' => 'technical',
+              'givenName' => 'root',
+              'surName' => 'root',
+            ),
+          ),
+          'certData' => 'MIID1TCCAr2gAwIBAgIJAJI03f54IC2GMA0GCSqGSIb3DQEBCwUAMIGAMQswCQYDVQQGEwJHQjETMBEGA1UECAwKU29tZS1TdGF0ZTEPMA0GA1UEBwwGTG9uZG9uMQ4wDAYDVQQKDAVBdmFkbzEOMAwGA1UEAwwFQXZhZG8xKzApBgkqhkiG9w0BCQEWHHZsYy5zeXN0ZW1AYXZhZG9sZWFybmluZy5jb20wHhcNMTcwNjA5MDk0MDEwWhcNMjcwNjA5MDk0MDEwWjCBgDELMAkGA1UEBhMCR0IxEzARBgNVBAgMClNvbWUtU3RhdGUxDzANBgNVBAcMBkxvbmRvbjEOMAwGA1UECgwFQXZhZG8xDjAMBgNVBAMMBUF2YWRvMSswKQYJKoZIhvcNAQkBFhx2bGMuc3lzdGVtQGF2YWRvbGVhcm5pbmcuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzKVlaeIXCVlZ0zpZHgOrEzNdLYaPJu68+oLhG5uAIIzXo2DiOthR5RvPnNlmt8t3CqPEzvsWUI5Nxv6FgoqirDUU8KAH79+CWbtXLpAfrgk0KKa5WneeSP6EHaciLH3X7qnH1MbjfyWTKH2abvL0f5P3BgfnfsNjOUgrHo0lhp994jpHfxkJBLN5WsiQmCAGgN87V9DzhgRY0wrvsA1OfYjkpRqBxJOHYeYIbMdu2rnfWmGynMP65uglCBZK1u1LNCf1vC5nUbNHY0uCios6eF9BeBTKdsxhD4YclbdU1iIa6L7GDaxVn4sU+89OuU/r2skQCbFL7fzYMIzw6diq2QIDAQABo1AwTjAdBgNVHQ4EFgQUFs5w4HRnLtvuAKqJ4r23DzaYAD8wHwYDVR0jBBgwFoAUFs5w4HRnLtvuAKqJ4r23DzaYAD8wDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAeo5DrJK1/XhVcmaTAahKECXFcxyYJncrPcXFuYC8Rg5rO9MvPetbPLkaT3WoWDZAhpILOLcB2kZKrY2mRYYZCXrSi5AZiF1pAoXqXkUErsmFP3Nk4onNb7VZpC0ll9sCCb97LWQMY+glV4bAsd2gCc07xIU4D2lU9lhHS0PZt9EKgKiE/b0DuDMnpJHr9SrsjklYltAloXJY0SjOiI3g45UxUIs7G54gtU/3OcQiuUSvC3DTUztW5/Zm4QaF37XMLVrb7jTqcFLh/ZrE5NbPrTPjuMS6m9UFSCKttyz0CnudtuAis4kP9i2TERIDJYo6/qV+iWCAbg2vhZ/I9hejKg==',
+        );
+      test_user_creds_01: "testavd01:TestAvocado01"
+      test_user_login_01: testavd01
+      test_user_firstname_01: Avado1
+      test_user_lastname_01: Avocado1
+      test_user_email_01: sascha.peter@avadolearning.com
+      test_user_creds_02: "testavd02:TestAvocado02"
+      test_user_login_02: testavd02
+      test_user_firstname_02: Avado2
+      test_user_lastname_02: Avocado2
+      test_user_email_02: sascha.peter@fusetech.io
```

Comment:
I've restructured the pillar and made the code iterate over the values. Will adjust the vagrant config for that as well shortly.